### PR TITLE
Atualiza dependências do projeto (incluindo Django 3)

### LINKED ISSUE #31: Atualizar o docker-compose para a versão 3.x
Reparei que estamos utilizando a versão 2.x do compose file do Docker. No entanto, a versão mais atual, e recomendada pelo Docker, é a versão 3.x (https://docs.docker.com/compose/compose-file/compose-versioning/).

Esta não me parece ser uma implementação difícil. Caso precisem de ajuda, estou à disposição.

### PR DESCRIPTION
**Pull request by cuducos**
Esse PR inclui diversas atualizações nas dependências do projeto, sendo a mais importante a do Django 2 para 3 (7eb30fed92339509b8207ad78edc672366e7ce36). Além disso:

* 48ada5610ceaaf38c261eefb1db1e684135c393a para dar mais previsibilidade aos _builds_ do Docker, sugeri especificar a imagem como sendo `python:3.7-slim-buster` o invés de `python:3`,  assim temos a garantia de:
    * ser Python 3.7, especificado no `README.md`
    * um Debian específico (`buster`)
    * e uma imagem menor (`slim`)
* 48ada5610ceaaf38c261eefb1db1e684135c393a talvez por essa falta de previsibilidade na versão da imagem do Docker, a instalação do `psycopg2-binary` estava falhando e foi corrigida
* 853460848c4a9610281788acfcfb73d4c15de6db atualizei a dependência `model-mommy` que agora se chama `model-bakery`
* 853460848c4a9610281788acfcfb73d4c15de6db atualizei as demais dependências do projeto para as versões atuais
* 1094b16 atualizei a versão das definições do Docker Compose<br> fix #31

### CODE REVIEW

**Review by jbiason — APPROVED** [review]

**Comment by jbiason** [comment]
Mando nada nessa firma, mas fiz o code review e parece tudo ok :+1:

**Comment by mazulo** [comment]
@cuducos Pergunta que pode parecer besta but better be safe: a atualização do Django foi major, testando local não quebrou nada? (ou os testes?). A pergunta é pq não temos testes mais rodando no CI do Travis. Ainda não migramos para GitHub Actions (aprendi isso esses dias, tava planejando fazer no final de semana)

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,13 @@
-FROM python:3
+FROM python:3.7-slim-buster
```
Comment (filipecifali) [inline Dockerfile:1]: Provavel que dê pra fazer bump pra `3.8-slim-bullseye`?

Comment (cuducos) [reply Dockerfile:1]: Se a documentação diz `>=3.7`, acho melhor niverlamos pela mais antiga (e a documentação diz isso no `README.md`).

Quando decidirmos que é hora de derrubar o suporte para 3.7, aí atualizamos, o que acha?

Em outras palavras: atualizar o Python pode ser assunto para um novo PR.

**Comment by cuducos** [comment]
> Pergunta que pode parecer besta

Nenhuma pergunta em code review é besta, na minha opinião 💜 

>  a atualização do Django foi major, testando local não quebrou nada? (ou os testes?)

Belo ponto! O que eu testei:

* `python manage.py check`
* `python manage.py test`
* `docker-compose up` (configurando o Postgres manualmente)

Nenhum erro no `check` ou `test`, e naveguei em algumas páginas e não vi erro. No entando, não conheço tanto os _edge cases da aplicação para saber se sou uma boa pessoa para esses testes manuais…

> Ainda não migramos para GitHub Actions (aprendi isso esses dias, tava planejando fazer no final de semana)

Vou registrar isso em uma issue, assim fica mais transparente para quem mais quiser/puder ajudar caso teu final de semana fique pequeno : )

File: Dockerfile (hunk 1 above)
Comment (cuducos) [reply Dockerfile:1]: (ah, nada contra o `bullseye`… só vou esperar ver o que acham dessa parte do 3.7 vs 3.8 e faço um novo commit!)

Comment (filipecifali) [reply Dockerfile:1]: Tudo bem por mim, como eu tava lendo a declaração da imagem apenas não notei que no REAME.md tava `3.7` porque a build da imagem poderia pegar superiores (alias fazendo a build hoje provavelmente usaria `3.10`)

Comment (filipecifali) [reply Dockerfile:1]: 🤦 acabei de perceber que eu tava pensando no outro projeto (https://github.com/pythonbrasil/associados)  e não nesse, ignora tudo que falei 😅

**Review by mazulo — APPROVED** [review]
Muito brigadão ❤️

**Review by filipecifali — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (cuducos) [reply Dockerfile:1]: Hahaha… vou pra esse na sequência
